ERCS/erc-7786.md

```diff
@@ -4,7 +4,8 @@ title: Cross-Chain Messaging Gateway
 description: An interface for contracts to send and receive cross-chain messages containing arbitrary data.
 author: Francisco Giordano (@frangio), Hadrien Croubois (@Amxx), Ernesto Garcia (@ernestognw), CJ Cobb (@cjcobb23), Sergey Gorbunov (@sergeynog), joxes (@Joxess)
 discussions-to: https://ethereum-magicians.org/t/erc-7786-cross-chain-messaging-gateway/21374
-status: Draft
+status: Last Call
```

Comment:
```suggestion
status: Review
```

Reply:
This ERC has (informally) been in review for months and publicly discussed in a few venues, including a dedicated L2 Interop WG call (https://github.com/ethereum/pm/issues/1439, [tweet](https://x.com/ellierdavidson/status/1909645927909843195)). Based on this, would you consider a direct move to Last Call?